Grammar: Optionally & type (#895)
withastro/docs: src/pages/en/core-concepts/layouts.md

## Markdown Layouts

> Page layouts are especially useful for [Markdown files](/en/guides/markdown-content/#markdown-pages). This file type can use `layout: (link to a file)` inside the fences to specify a layout component to wrap this file in HTML.

Reviewer: ```suggestion
Page layouts are especially useful for [Markdown files](/en/guides/markdown-content/#markdown-pages). Markdown files can use the special `layout` frontmatter property to specify a `.astro` component to use as a page layout.
```

> When a markdown file specifies a layout file, it passes a single `{ content }` property back to the layout file. Included in the contents are all the fenced in properties and the final HTML output of the page.

Reviewer: ```suggestion
When a Markdown file includes a layout, it passes a single `{ content }` property to the layout file which includes the frontmatter properties and the final HTML output of the page.
```

> See the above example for how to pass content properties of a markdown file to a layout file.

Reviewer: ```suggestion
See the above example for how to pass content properties of a Markdown file to a layout.
```

```markdown

> // Example src/pages/posts/post-1.md

Reviewer: ```suggestion
// src/pages/posts/post-1.md
```
